src/Vocabulary/VInput.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+    <div :class="[ 'control', sizeClass, { 'has-icons-left': hasLeftIcon } ]">
+        <label :class="sizeClass">
+            <span
+                v-if="label"
+                :class="['label', { 'label-bold': isLabelBold}]"
+            >{{ label }}</span>
+            <span
+                v-if="description"
+                class="description"
+            >{{ description }}</span>
+            <span class="control-inner">
+                <span
+                    v-if="hasLeftIcon"
+                    class="icon is-left"
+                >
+                    <slot name="left-icon" />
+                </span>
+                <input
+                    v-if="!isTextArea"
+                    class="input"
+                    :class="[sizeClass]"
+                    :placeholder="placeholder"
+                    :disabled="isDisabled"
+                    :type="type"
+                    :value="localValue"
+                    @input="onInput"
+                >
+                <textarea
+                    v-if="isTextArea"
+                    class="textarea"
+                    :value="localValue"
+                    :placeholder="placeholder"
+                    :disabled="isDisabled"
+                    :readonly="isReadonly"
+                    @input="onInput"
+                />
+                <span
+                    v-if="hasRightIcon"
+                    class="icon is-right"
+                >
+                    <slot name="right-icon" />
+                </span>
+            </span>
+        </label>
+    </div>
+</template>
+<script>
+export default {
+    name: 'VInput',
+    props: {
+        value: {
+            type: String,
+            default: ''
+        },
+        type: {
+            type: String,
+            default: 'text'
+        },
+        label: {
+            type: String,
+            default: ''
+        },
+        description: {
+            type: String,
+            default: ''
+        },
+        placeholder: {
+            type: String,
+            default: ''
+        },
```

Comment:
You can use `inheritAttrs: false` and then place the `$attrs` on the input. See [docs](https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance).

Reply:
Sorry for a late reply, @dhruvkb ! I was writing this component as a more general Input component for Vocabulary that can be a text input or a text area.  The solution you propose is really clean, but for a simpler component.

Some of these props, such as `description`, `label`, `value`  are necessary for the component itself, and some could be bound to the `input` or the `textarea`, so I thought leaving them as props would be better.